xc7/archs/artix7.cmake

```diff
@@ -0,0 +1,13 @@
+set(ROI_PART xc7a35tcpg236-1)
```

Comment:
Move these directly into the add_arch_define to avoid variable name collisions?

Reply:
Same with the zync version

Reply:
Ok